src/theory/arith/nl/nonlinear_extension.cpp

```diff
@@ -860,6 +860,85 @@ void NonlinearExtension::presolve()
   Trace("nl-ext") << "NonlinearExtension::presolve" << std::endl;
 }
 
+void NonlinearExtension::runStrategy(Theory::Effort effort,
+                                     const std::vector<Node>& assertions,
+                                     const std::vector<Node>& false_asserts,
+                                     const std::vector<Node>& xts)
+{
+  if (!d_strategy.isStrategyInit())
```

Comment:
Nit: we could probably put this in a better place, e.g. a finishInit method called from TheoryArith::finishInit.